### DISCUSSION TITLE
支持电视下载和使用吗？

### DISCUSSION THREAD

**Discussion by ohhhhhe** [open-ani/animeko#86] [General]
如题

**Comment by Him188** [comment]
安卓系统就行, 但是我没有电视设备也没有使用需求所以我不会专门做适配. 如果装上能用最好, 不能用的话欢迎 PR

**Reply by Him188** [reply]
重新想了一下, 理论上是能直接支持的, 电视遥控器应该是通过移动焦点来控制的, 我的 UI 应该都是支持的.
就是目前只有手机 UI, 电视这种横屏的很有可能会有问题. 等未来做一个平板 UI 应该就没问题了.

**Reply by ohhhhhe** [reply]
好的，谢谢大佬的解答
